### ISSUE TITLE
no protocol

### ISSUE THREAD

**Issue by isom3250** [icy/google-group-crawler#5]
Hi Himanshu:

I have tried your code and found the following error. Should we use URI? URL seems to have a bug.

Exception in thread "main" java.net.MalformedURLException: no protocol: 'https
/groups.google.com/forum/#!forum/ibm.software.websphere.application-server'

Thank you very much!

**Comment by icy** [comment]
I guess you call my script from a `Java` program. If that's the case, you need to set up some environments (see `README.md`). The most important variable is `_GROUP`; it should be `/ibm.software.websphere.application-server`. 

Please refer to you language specification to set up a variable for a sub-shell invocation.

Thanks,

**Comment by isom3250** [comment]
Hi:

Thanks for your fast response. I have added an environment variable _GROUP and I still have the following no protocol error:

C:\maven\googleGroup\google-grp-scraper\target>java -jar google-grp-scraper-0.0.
1-SNAPSHOT.jar 'https://groups.google.com/forum/#!forum/ibm.software.websphere.a
pplication-server' 20
Exception in thread "main" java.net.MalformedURLException: no protocol: 'https:/
/groups.google.com/forum/#!forum/ibm.software.websphere.application-server'
at java.net.URL.(Unknown Source)
at java.net.URL.(Unknown Source)
at java.net.URL.(Unknown Source)
at com.pramati.scraper.google_grp_scraper.ScraperStartup.main(ScraperSta
rtup.java:9)

Thanks again. (I posted a new issue but I guess this is the right place to continue the dialogue. sorry.)
